src/index.ts

```diff
@@ -43,39 +48,78 @@ async function run(): Promise<void> {
     const registryPathList: string[] = [];
 
     // check if image with all the required tags exist in Podman image storage
-    const isPresentInPodman: boolean = await checkImageInPodman();
+    const podmanImageStorageCheckResult: ImageStorageCheckResult = await checkImageInPodman();
+
+    const podmanFoundTags: string[] = podmanImageStorageCheckResult.foundTags;
+    const podmanMissingTags: string[] = podmanImageStorageCheckResult.missingTags;
+
+    if (podmanFoundTags.length > 0) {
+        core.info(`Tag(s) ${podmanFoundTags.join(", ")} of ${imageToPush} found in Podman image storage`);
+    }
+
+    if (podmanMissingTags.length > 0) {
```

Comment:
this is not quite in line with the design we discussed -

The problem here is that `imageToPush:<any tag>` being missing in podman is not a warning, because the tags may all be in Docker and that's an expected, success case.

I would wait to produce any output until after checking both docker and podman.

Reply:
Yes, I agree.
May be we can log only what all tags are found, and log the error if tags are missing from both.
Like: 
```
Tag(s) v1 not found in podman image storage and Tag(s) latest not found in docker image storage
```

I only logged it as warning, thinking of use case where user thinks that he build from podman but tags are not found in podman, in such as case I think warning makes sense.

Reply:
> I only logged it as warning, thinking of use case where user thinks that he build from podman but tags are not found in podman, in such as case I think warning makes sense.

This makes sense, if some of the tags are found in podman but not all tags.

It does not make sense if the tags are all found in docker and none are in podman, in that case there should be no warning.

Reply:
Yes, makes sense. Will change this accordingly.